src/xrpld/app/tx/detail/Clawback.cpp

```diff
@@ -51,8 +60,46 @@ Clawback::preflight(PreflightContext const& ctx)
     return preflight2(ctx);
 }
 
+template <>
+NotTEC
+preflightHelper<MPTIssue>(PreflightContext const& ctx)
+{
+    if (!ctx.rules.enabled(featureClawback))
```

Comment:
should it check for the `MPTokenV1` amendment as well?

Reply:
It's checked a few lines below. It could be combined.

Reply:
nevermind, it's checked on line 73, but the two checks can be part of one if statement